src/main/kotlin/com/markskelton/settings/ThemeSettingsUI.kt

```diff
@@ -0,0 +1,132 @@
+    private const val REPOSITORY = "https://github.com/one-dark/jetbrains-one-dark-theme"
+    private val CHANGELOG_URI = URI("$REPOSITORY/blob/master/CHANGELOG.md")
+    private val ISSUES_URI = URI("$REPOSITORY/issues")
+    private val MARKETPLACE_URI = URI("https://plugins.jetbrains.com/plugin/11938-one-dark-theme")
+  }
+
+  override fun getId(): String = "com.markskelton.ThemeSettings"
+
+  override fun getDisplayName(): String =
+    THEME_SETTINGS_DISPLAY_NAME
+
+  private val initialThemeSettingsModel = constructSettingModel()
+
+  private val themeSettingsModel = initialThemeSettingsModel.copy()
+
+  override fun isModified(): Boolean {
+    return initialThemeSettingsModel != themeSettingsModel
+  }
+
+  override fun apply() {
+    persistChanges()
+    ApplicationManager.getApplication().messageBus.syncPublisher(
+      THEME_CONFIG_TOPIC
+    ).themeConfigUpdated(ThemeSettings.instance)
+  }
+
+  private fun persistChanges() {
+    registerSettingsChange(themeSettingsModel.isBold, {
+      ThemeSettings.instance.isBold
+    }) {
+      ThemeSettings.instance.isBold = it
+    }
+    registerSettingsChange(themeSettingsModel.isVivid, {
+      ThemeSettings.instance.isVivid
+    }) {
+      ThemeSettings.instance.isVivid = it
+    }
+    registerSettingsChange(themeSettingsModel.isItalic, {
+      ThemeSettings.instance.isItalic
+    }) {
+      ThemeSettings.instance.isItalic = it
+    }
+  }
+
+  override fun createComponent(): JComponent? =
+    createSettingsPane()
+
+  private fun createSettingsPane(): DialogPanel =
+    panel {
+      titledRow("Main Settings") {
+//        row {
+//          cell {
+//            checkBox(
+//              "Bold Characters",
+//              themeSettingsModel.isBold,
+//              comment = "Uses bold fonts for certain language keywords",
+//              actionListener = { _, component ->
+//                themeSettingsModel.isBold = component.isSelected
+//              }
+//            )
+//          }
+//        }
+        row {
+          cell {
+            checkBox(
+              "Italic Characters",
+              themeSettingsModel.isItalic,
+              comment = "Uses italic font for language keywords and comments",
+              actionListener = { _, component ->
+                themeSettingsModel.isItalic = component.isSelected
+              }
+            )
+          }
+        }
+        row {
+          cell {
+            checkBox(
+              "Vivid Pallette",
+              themeSettingsModel.isVivid,
+              comment = "Uses the One-Dark vivid color pallette",
+              actionListener = { _, component ->
+                themeSettingsModel.isVivid = component.isSelected
+              }
+            )
+          }
+        }
+      }
+      titledRow("Miscellaneous Items") {
+        row {
+          cell {
+            button("View Issues") {
+              browse(ISSUES_URI)
+            }
```

Comment:
Could we have this open the new issue URL? https://github.com/one-dark/jetbrains-one-dark-theme/issues/new

```suggestion
            button("Report an Issue") {
              browse(ISSUES_URI)
            }
```

Reply:
I thought about that. My thought was this gives them an opportunity to see if the issue they are about to file has already been reported.

Reply:
Good point.  I guess I would recommend taking the best of both worlds where the button says Report an Issue which is a very clear call to action but have the action button go to the issues list like you have it.